package.json

```diff
@@ -134,7 +134,6 @@
   },
   "husky": {
     "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
```

Comment:
Turning this off as we said in the handbook. You can still use `yarn commit` but its not required.